### PR TITLE
[Refactor][Manifest] promote RMSNormFwdOp to implemented

### PR DESCRIPTION
**Pull request by lcy-seso**
## Changes

- `tileops/ops_manifest.yaml`: `RMSNormFwdOp.status: spec-only → implemented`. Remove now-stale comment.

Code already conforms to the manifest spec (PR #1061). Per trust model, only `align-op@FLIP_STATUS` flips status — hence a separate one-line PR.

## Tests

- [x] `python scripts/validate_manifest.py --check-op RMSNormFwdOp` — passes.
- [x] `pytest tests/ops/test_rms_norm.py` (smoke + full) — 18/18 pass on H200, including tail-M (1025×4096) and dim=1 cases.
- [x] `pytest benchmarks/ops/bench_rms_norm.py -k llama-3.1-8b-decode-bfloat16` — produces numbers (M=1 decode shape).

### CODE REVIEW

File: tileops/ops_manifest.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ ops:
   RMSNormFwdOp:
     ref_api: "torch.nn.functional.rms_norm"
     family: normalization
-    status: spec-only  # impl lacks dim param; fix in follow-up PR
+    status: implemented
```
[the remark this answers is not in the document]
Comment (lcy-seso) [reply tileops/ops_manifest.yaml:47]: Rejecting. The kernel does handle tail-M correctly — partial-block masking is provided by tilelang's `T.copy` and `T.Parallel` at the runtime layer, not in the Python source. Evidence:

1. `tests/ops/test_rms_norm.py` has a `pytest.mark.full` parameter set `(1025, 4096, …)` literally annotated `# Tail-M: M not divisible by block_m (proves T.copy partial block safety)`. This test passes on H200.
2. PR #1063's `/align-op` run included `benchmarks/ops/bench_rms_norm.py::test_rms_norm_bench[llama-3.1-8b-decode-bfloat16]` which is the M=1 decode shape — passed.
3. Just re-ran `pytest -m full`: 10/10 pass including the `1025-4096` tail-M case for both fp16 and bf16.

Specifically the kernel's `T.copy(x[pid_m * block_m, 0], shared_buf)` translates to a tilelang LDGSTS / cp.async with bounds-aware predication; the same applies to `T.Parallel(block_m, N_padded)`. The bounds check is in tilelang IR rewrite, not user code.

**Review by Ibuki-wind — APPROVED** [review]
Clean manifest-only status flip. `RMSNormFwdOp` now matches the manifest-facing constructor and source wiring, and targeted validation passes with only the known staged-rollout parity warnings.
